next/tailwind.config.js

```diff
@@ -40,6 +40,13 @@ module.exports = {
     // These properties *extend* the defaults:
     extend: {
       colors: {
+        steelgray: "#221f2d",
+        scarpaflow: "#555161",
```

Comment:
![image](https://user-images.githubusercontent.com/185412/222728095-81836b47-49d3-441c-8a1b-3c099a4ed641.png)

(Scarpa = shoe in Italian, "shoe flow" :D )

I agreed with Mikko to change this to scapaflow :)